docs/competency-api/documentation.md

```diff
@@ -5,4 +5,94 @@ nav_order: 1
 parent: Condition
 grand_parent: Competency API
 ---
-# Documentation
\ No newline at end of file
+# Documentation
+
+## Description
+
+### Headers for all requests
+```json
+{
+    Bearer: "Bearer Example.bearer.token"
```

Comment:
The header is authorization not Bearer. The value of the req.header.authorization is "Bearer eyasdf.asdfa.asdf"